src/covid19model/data/parameters.py

```diff
@@ -54,27 +54,30 @@ def get_COVID19_SEIRD_parameters():
     -----------
     parameters = get_COVID19_SEIRD_parameters()
     """
+    abs_dir = os.path.dirname(__file__)
+    rel_dir = os.path.join(abs_dir, "../../../data/raw/model_parameters")
```

Comment:
Can you do a merge of master, as I think this already changed

Reply:
Yes, I did update this morning and merged with the adjustments Jenna did earlier.